mtags-shared/src/main/scala/scala/meta/internal/metals/TimeFormatter.scala

```diff
@@ -0,0 +1,16 @@
+package scala.meta.internal.metals
+
+import java.text.SimpleDateFormat
+import java.util.Calendar
+
+import scala.util.Try
+import scala.util.matching.Regex
+
+object TimeFormatter {
+  private val format = new SimpleDateFormat("yyyy-MM-dd_HH-mm-ss-SSS")
```

Comment:
True. I don’t have a good answer. You could print months as letters to avoid confusion. Best print some names using different options and see what looks most readable/informative to you.